Domains/0-Core/BisCore.ecschema.xml

```diff
@@ -877,6 +896,33 @@
             <ClassHasHandler xmlns="BisCore.01.00.00"/>
         </ECCustomAttributes>
     </ECEntityClass>
+    <ECEntityClass typeName="TemplateRecipe3dInPlan" displayLabel="Plan-Projection Template Recipe" description="A recipe that uses a Plan-Projection template for creating new instances.">
+        <BaseClass>RecipeDefinitionElement</BaseClass>
+        <BaseClass>ISubModeledElement</BaseClass>
+    </ECEntityClass>
+
+    <ECEntityClass typeName="DefinitionThumbnail" modifier="None" displayLabel="Thumbnail" description="A bis:DefinitionElement that captures the Thumbnail of one or more bis:DefinitionElement instances.">
+        <BaseClass>DefinitionElement</BaseClass>
+        <ECProperty propertyName="Image" typeName="binary" displayLabel="Image" description="Image of the Thumbnail.">
+            <ECCustomAttributes>
+                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
+            </ECCustomAttributes>
+        </ECProperty>
+        <ECProperty propertyName="ImageFormat" typeName="string" displayLabel="Image Format" description="Format of the Image captured as MIME-type.">
+            <ECCustomAttributes>
+                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03"/>
+            </ECCustomAttributes>
+        </ECProperty>
+    </ECEntityClass>

```

Comment:
I know a thumbnail is a different use case but we should consider how this compares to how we represent textures.

https://github.com/iTwin/bis-schemas/blob/cdb0067a686d619074b856799ad725affbcf0e66/Domains/0-Core/BisCore.ecschema.xml#L2281

The referenced ImageSourceFormat enum in iTwin.js supports jpg, png and svg: https://github.com/iTwin/itwinjs-core/blob/9b71eebc6663188ae999f9c8d919f278660048ae/core/common/src/Image.ts#L105

Reply:
Introduced the supported formats in both cases as enums.